src/Databases/MySql.php

```diff
@@ -113,6 +116,16 @@ public function skipLockTables()
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function useColumnStatisticsOff()
```

Comment:
Rename the function and variable to something more human like `doNotUseColumnStatistics`.

Reply:
Done!